editor/src/main/com/mbrlabs/mundus/editor/Editor.kt

```diff
@@ -170,12 +183,20 @@ class Editor : Lwjgl3WindowAdapter(), ApplicationListener,
                 debugRenderer.end()
             }
 
-            Gdx.gl.glLineWidth(globalPreferencesManager.getFloat(MundusPreferencesManager.GLOB_LINE_WIDTH_HELPER_LINE, MundusPreferencesManager.GLOB_LINE_WIDTH_DEFAULT_VALUE))
-            scene.batch.begin(scene.cam)
-            context.helperLines.render(scene.batch)
-            scene.batch.end()
-            Gdx.gl.glLineWidth(MundusPreferencesManager.GLOB_LINE_WIDTH_DEFAULT_VALUE)
-
+            val renderExtensions = pluginManager.getExtensions(RenderExtension::class.java)
+            if (renderExtensions.isNotEmpty()) {
+                renderExtensions.forEach {
+                    scene.batch.begin(scene.cam)
```

Comment:
Could batch begin and end be moved out of the forEach, so that we batch the render extensions together. Thoughts? I haven't tested it.

Reply:
If I move out the batch begin and end from the foreach then the different line width will not work. The user can set different width for lines in helper lines plugin. The Editor sets line width to default value because a plugin should not affect other plugins or Mundus Editor's behaviour. This is why I added batch begin and end into foreach.

Reply:
Got it, that makes sense